zipkin-web/src/main/resources/app/js/component_ui/trace.js

```diff
@@ -287,7 +287,9 @@ define(
             $.each(self.parents[id], function(i, pId) { self.spans[pId].openChildren += 1; });
           });
           $.each(self.spansByService, function(svc, spans) {
-            $.each(spans, function(i, $span) { $span.inFilters += 1; });
+            $.each(self.getSpansByService(svc), function(i, $span) {
```

Comment:
Yes, `getSpansByService(svc)` basically returns `spans` (the argument from L289), _except_ when `spans` is a list of strings - then it will convert the list of strings into a jquery object, before returning that jquery object. (like you can see in its implementation)